.github/workflows/nouveau.yml

```diff
@@ -0,0 +1,20 @@
+name: Run Gradle on PRs
+on: pull_request
+jobs:
+  gradle:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macos-latest ]
```

Comment:
Hey, I think there is no need to exclude Windows here :wink:, see https://github.com/gradle/gradle-build-action#use-the-action-to-setup-gradle

Reply:
I excluded it because it _failed_ when I tried it.